Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -1016,10 +1016,13 @@ bool RenderStyle::changeRequiresLayout(const RenderStyle& other, OptionSet<Style
 #if ENABLE(TEXT_AUTOSIZING)
             || m_inheritedData->specifiedLineHeight != other.m_inheritedData->specifiedLineHeight
 #endif
-            || m_inheritedData->fontCascade != other.m_inheritedData->fontCascade
             || m_inheritedData->horizontalBorderSpacing != other.m_inheritedData->horizontalBorderSpacing
             || m_inheritedData->verticalBorderSpacing != other.m_inheritedData->verticalBorderSpacing)
             return true;
+
+        if (m_inheritedData->fontData != other.m_inheritedData->fontData
+            && m_inheritedData->fontData->fontCascade != other.m_inheritedData->fontData->fontCascade)
```

Comment:
`DataRef::operator==` will already compare the contents of the pointed to object, after checking the pointer itself, so this should just be:

```
if (m_inheritedData->fontData != other.m_inheritedData->fontData) {
    ...
```